README.md

```diff
@@ -13,6 +13,8 @@ This protocol can be configured via Kuzzle'rc configuration, under `server > pro
 | ``port`` | ``1883`` | Integer > 1024 | Network port to open 
 | ``requestTopic`` | ``"Kuzzle/request"`` | String | Name of the topic listened by the plugin for requests 
 | ``responseTopic`` | ``"Kuzzle/response"`` | String | Name of the topic clients should listen to get requests result 
+| ``developmentMode`` | `false` | Boolean | Switches `responseTopic` back to a regular public topic
```

Comment:
nitpicking: maybe a bit too verbose, why not `development`?

Reply:
I agree this is bit verbose, which I consider not being a real problem considering the induced risk.

I went forth and back between `debug` and `development` semantics during implementation but to me, debug is related to getting some (more) verbose outputs, which is not the case here.
`development` alone is also too vague imo, so I ended up with this.